modules/imaging_uploader/test/TestPlan.md

```diff
@@ -102,10 +102,35 @@
     Check that an appropriate message is written in the Console Output (i.e. "`The PatientName read from the DICOM header
     does not start with MTL0002_222222_V01 from the mri_upload table`").
     [Manual Testing]
-19. First, set the config setting 'ImagingUploader auto launch' to 'Yes'. Then, edit your prod file (in
+19. Set the config setting 'ImagingUploader auto launch' to 'Yes'. Then, edit your prod file (in
 	<LORIS MRI code dir>/dicom-archive/.loris-mri) and comment out the definition of the @db array. Once these operations
 	are done, upload any valid scan: check in the server processes manager that this fails with an error code of 4.
 	Now try to upload the scan again. When the system asks you if you want to overwrite the existing 
 	archive, answer 'Yes'. Check that this reupload fails with error code 4 (and not 2). 
 	Related to Redmine#14093 and PR#3555.
 	[Manual Testing]
+20. Set the config setting 'ImagingUploader auto launch' to 'No'. Upload an MRI archive that can successfully be
+    uploaded. Check that the value of the Progress column for the uploaded archive is 'Not started'.
+    [Manual Testing]
+21. Set the config setting 'ImagingUploader auto launch' to 'Yes'. Upload an MRI archive that can successfully be
+    uploaded and that will be processed successfully by the pipeline. Once the upload is finished and processing of the
+    archive has begun, check that the value of the Progress column for the uploaded archive is 'In Progress...'.  Click 
+    on the uploaded archive in the results table and select 'Detailed' in the combo box of the log panel. Check that an 
+    increasing serie of dots is displayed along with a rotating bar to indicate that the archive is being processed. Once
+    processing is finished, reload the page and check that the value of the Progress column for the uploaded archive is 
+    'Success (X out of Y)', where X is the number of MINC files successfully inserted and Y is the total number of MINC 
+    files created. 
+    [Manual Testing]
+23. Set the config setting 'ImagingUploader auto launch' to 'Yes'. Upload a file that can successfully be uploaded but that 
```

Comment:
Although GitHub auto adjust number, it should be properly written.

Reply:
@PapillonMcGill I don't understand your comment

Reply:
She means that your list went from 21 to 23 and skipped 22.